### PR TITLE
Satisfaction 75 turn off backlight on suspend, restore config on wakeup

### PR DESCRIPTION
**Pull request by MarMed**
## Description

Made changed to led.c to turn off backlight on host sleep, and restore user configuration on wakeup

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
@awkannan Any issues with this?

File: keyboards/cannonkeys/satisfaction75/led.c [hunk 1]
```diff
@@ -80,6 +80,21 @@ void backlight_init_ports(void) {
   }
 }
 
+void suspend_power_down_user(void) {
+    backlight_set(0);
+}
+void suspend_wakeup_init_user(void) {
+  if(kb_backlight_config.enable){
+    if(kb_backlight_config.breathing){
```
Comment (fauxpark) [inline keyboards/cannonkeys/satisfaction75/led.c:88]: If suspend messes up backlight breathing then that is a bug in the backlight feature and should be fixed there instead.

Comment (MarMed) [reply keyboards/cannonkeys/satisfaction75/led.c:88]: I am hardly an expert, I am just going by what I've gleamed from looking over the led.c, but I believe the backlight and backlight breathing is implemented separate from the base qmk code in order for it to be configured from the OLED? I could be totally wrong. My change just sets the backlight to 0 on suspend, and then on wakeup restores the previous setting by executing part of backlight_init_ports.

Comment (fauxpark) [reply keyboards/cannonkeys/satisfaction75/led.c:88]: Ah, this board has a custom backlight implementation. I'm not sure if that's necessary though...

Comment (MarMed) [reply keyboards/cannonkeys/satisfaction75/led.c:88]: Yeah, possibly, but that's beyond me.

**Review by tzarc — APPROVED** [review]

**Pull request merged**